cli/args.go

```diff
@@ -218,7 +229,7 @@ func parseTerragruntOptionsFromArgs(terragruntVersion string, args []string, wri
 	opts.ModulesThatInclude = modulesThatInclude
 	opts.StrictInclude = strictInclude
 	opts.Check = parseBooleanArg(args, optTerragruntCheck, os.Getenv("TERRAGRUNT_CHECK") == "true")
-	opts.HclFile = filepath.ToSlash(terragruntHclFilePath)
```

Comment:
Not sure if `ToSlash()` should be avoided, it is converting paths to the same format, and without `ToSlash()` handling of paths in Windows may not work.

Reply:
Good catch, I've added a `CleanPaths` func that does the work.